tasks/hivemind/classify_question.py

```
@@ -38,21 +55,22 @@
         )
 
         response_text = response.choices[0].message.content.strip().lower()
 
-        if "true" in response_text:
-            return True
-        elif "false" in response_text:
-            return False
+        if re.match(r"^(?:0|1|\d+\.\d+)$", response_text):
+            return float(response_text) >= self.rag_threshold
         else:
             raise ValueError(f"Wrong response: {response_text}")
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)